gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
This file and all below files are standard, right?

Reply:
Absolutely! It has been automatically created when I ran ```./gradlew wrapper```